index.js

```diff
@@ -170,6 +175,11 @@ const fields = [
 ]
 
 const defaultSortOrder = fields.map(({ key }) => key)
+const overFields = pipe(
+  fields
+    .filter(({ over }) => over)
+    .map(({ key, over }) => overProperty(key, over)),
```

Comment:
a filter + map could be combined to a reduce.

Reply:
fixed